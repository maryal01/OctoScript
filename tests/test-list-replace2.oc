list<int> garr = [1, 2, 3, 4, 5, 6, 7];
fn test_function (list<int> arr) -> list<int> {
    list<int> arr1 = arr.replace(2, 31);
    list<int> arr2 = arr1.replace(3, 41);
    return arr2;
}

list<int> rvalue = test_function(garr);

int i = 0;
int length = garr.length();
while(i < length){
    print("%d,", garr.get(i));
    i = i + 1;
}