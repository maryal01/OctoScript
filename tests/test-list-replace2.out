1,2,31,41,5,6,7,